src/App.css

```diff
@@ -21,29 +21,40 @@
   background-color: var(--dark-color);
 }
 
-header {
-  background-color: var(--light-accent-color);
-  color: var(--dark-color);
-}
-
 nav {
   background-color: var(--dark-accent-color);
 }
 
-.Link {
-  padding: 10px;
-  color: var(--light-accent-color);
-}
-
 img {
-  width: 100%;
+  width: 80%;
   height: 200px;
```

Comment:
I'd say you may even want to make the image width smaller as it is very stretched and distorted on bigger screens!